dataland-frontend/tests/e2e/fixtures/frameworks/gdv/GdvPreparedFixtures.ts

```diff
@@ -122,3 +133,22 @@ function modifyPreparedFixturesPartTwo(newFixture: FixtureData<GdvData>): void {
       YesNo.Yes;
   }
 }
+
+/**
+ * Creates the contents for a time series based on a current year value and sets null for all numbers
+ * @param currentYear is the base year for the time series
+ * @returns the map of years to time series data objects
+ */
+function createEnergieverbrauchWithNullNumbers(currentYear: number): {
+  [p: string]: BerichterstattungEnergieverbrauchValues;
+} {
+  const objectToReturn: { [p: string]: BerichterstattungEnergieverbrauchValues } = {};
+  for (let i = -3; i <= 3; i++) {
+    objectToReturn[(currentYear + i).toString()] = {
```

Comment:
In the backend it gets mapped to a map<Int, whatever> but here the keys here are strings. Is this a problem when posting the data?

Reply:
No.
JS handles that map as an object like this:

``{ 
2021: ...
2022: ...
}``

So if I want to do an entry for 2023 into this "map", I need to do

`mapName["2023"] = ...`

For TS/JS the key is a string, and that is why I need to cast the number to a string here.


If you post this dataset and this object/JSON reaches our backend, it will deserialize this object to a `Map<Int, WhateverObject>`
As long as the key in the object is castable to an Int for Kotlin, you will have no problems.

Reply:
Cool